### PR TITLE
Remove organization logo from the embed/iframe cards

### PR DESCRIPTION
**Pull request by jarvisct**
#### :tophat: What? Why?
Remove organization logo from the embed/iframe cards.

#### :pushpin: Related Issues
- Related to #?

#### :clipboard: Subtasks
- [x] Add `README` entry

### :camera: Screenshots (optional)
Description

### CODE REVIEW

**Comment by andreslucena** [comment]
Hi. I have a doubt with the idea of "from email"?
As I see on the PR this is for the widgets/iframe/embed feature.

**Comment by jarvisct** [comment]
> Hi. I have a doubt with the idea of "from email"?
> As I see on the PR this is for the widgets/iframe/embed feature.

**Comment by jarvisct** [comment]
> Hi. I have a doubt with the idea of "from email"?
> As I see on the PR this is for the widgets/iframe/embed feature.

Hi @andreslucena, I change the pull request description.

**Comment by jarvisct** [comment]
Hi again @andreslucena,

I removed organization logo in general (see description). Could you confirm if this is the requeriment?

**Comment by andreslucena** [comment]
Hi @jarvisct 
#285 is already closed, as we've already have the changes on the design that we need. Do you have a JIRA/issue number on this issue? 
What we need and we've commented on the JIRA issue is to not have the logo ONLY on the embed/iframe card, as we need to integrate to other website - ie #271 https://www.decidim.barcelona/consells/horta.html

**Comment by jarvisct** [comment]
> Hi @jarvisct
> #285 is already closed, as we've already have the changes on the design that we need. Do you have a JIRA/issue number on this issue?
> What we need and we've commented on the JIRA issue is to not have the logo ONLY on the embed/iframe card, as we need to integrate to other website - ie #271 https://www.decidim.barcelona/consells/horta.html

Hi @andreslucena !

I don't have the issue number (JIRA). I have just applied the wanted changes.

**Comment by andreslucena** [comment]
@jarvisct please check DECIDIM-23

**Comment by andreslucena** [comment]
I've seen the changes. LGTM :+1:

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ This is the opensource code repository for "decidim-barcelona", based on [Decidi
 
 ## Upgrade notes
 
+- `app/views/layouts/decidim/widget.html.erb` modify the embed to match the current card view.
```
Comment (tramuntanal) [inline README.md]:
```suggestion
- `app/views/layouts/decidim/widget.html.erb` modify the embed to remove organization's logo.
```

**Pull request merged**